### PR TITLE
use linux not linux64

### PR DESCRIPTION
**Pull request by CJ-Wright**
In the selectors, this should make the bot happy, I think.

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ build:
 
 requirements:
   run:
-    - gmp  # [linux64]
-    - zlib  # [linux64]
+    - gmp  # [linux]
```
[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipe/meta.yaml]: Hmm ok the render that the bot does misses these lines. I'm not certain why. @msarahan thoughts? The bot is setting the arch to Linux during the render step but maybe that isn't enough?

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: Note that this was working before, so something changed that made this selector invalid now.

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipe/meta.yaml]: What do you mean? Has the bot ever sent a PR to this feedstock?

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipe/meta.yaml]: > What do you mean? Has the bot ever sent a PR to this feedstock?

Yep, see https://github.com/conda-forge/pandoc-feedstock/pulls?q=is%3Apr+author%3Aregro-cf-autotick-bot+sort%3Aupdated-desc

This feedstock was the "model" for the multiple sources problem BTW :smile:

[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipe/meta.yaml]: ah, ok I think this is fixed in https://github.com/regro/cf-scripts/pull/412

**Comment by CJ-Wright** [comment]
Closing, this was a problem with the bot using the wrong keyword in the config, the fix for that is in PR.

**Pull request closed without merging**